src/cli/com/yetanalytics/datasim/main.clj

```diff
@@ -152,9 +152,8 @@
                                   (if (= -1 override-seed)
                                     (.nextLong (Random.))
                                     override-seed)))]
-            (if-let [spec-error (input/validate input)]
-              (bail! [(binding [s/*explain-out* expound/printer]
-                        (expound/explain-result-str spec-error))])
```

Comment:
I like that the ugly `binding` is finally gone.

Reply:
me too